catowseriOS/catowser/BrowserContent/TopSites/TopSitesLegacyView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TopSitesLegacyView.swift
+//  catowser
+//
+//  Created by Andrei Ermoshin on 4/23/23.
+//  Copyright © 2023 andreiermoshin. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+struct TopSitesLegacyView: UIViewControllerRepresentable {
+    private let vm: TopSitesViewModel
+    typealias UIViewControllerType = UIViewController
+    
+    init(_ vm: TopSitesViewModel) {
+        self.vm = vm
+    }
+    
+    private var vcFactory: ViewControllerFactory {
```

Comment:
this computed property could be moved to some new protocol which extends system's `UIViewControllerRepresentable ` because it is used by every UIKit wrapper view